app/src/main/java/com/hamdan/forzenbook/data/repository/LoginRepository.kt

```diff
@@ -2,6 +2,6 @@ package com.hamdan.forzenbook.data.repository
 
 interface LoginRepository {
 
-    suspend fun getToken(email: String, password: String): LoginData?
+    suspend fun getToken(email: String): LoginData?
```

Comment:
What is this returning?

Reply:
well its giving back a data class that just holds a string which is currently the token, obviously I have to make a second function for the get and post as the post is the one that actually gets a response and it needs both the email and a validation code but thats out of scope of this ticket, I will follow up on this in another ticket

Reply:
so long story short it should return nothing for that one and be renamed